source/fuzz/fuzzer_pass_construct_composites.cpp

```diff
@@ -172,7 +158,8 @@ void FuzzerPassConstructComposites::RecordAvailableInstruction(
   if (type_id_to_available_instructions->count(inst->type_id()) == 0) {
     (*type_id_to_available_instructions)[inst->type_id()] = {};
   }
-  type_id_to_available_instructions->at(inst->type_id()).push_back(inst);
+  type_id_to_available_instructions->at(inst->type_id())
+      .push_back(inst->result_id());
```

Comment:
You can replace all this with `(*long_type_id_map_name)[inst->type_id()].push_back(inst->result_id())`.

Reply:
Done.